Makefile

```diff
@@ -147,10 +155,8 @@ PRIVILEGE_LEVEL?=
 check: lint test go-test
 
 .PHONY: test
-test: stacker $(REGCLIENT) $(SKOPEO) $(ZOT)
-	sudo -E PATH="$$PATH" \
-		LXC_BRANCH=$(LXC_BRANCH) \
-		LXC_CLONE_URL=$(LXC_CLONE_URL) \
+test: stacker $(REGCLIENT) $(SKOPEO) $(ZOT) $(BATS)
```

Comment:
because test is a .PHONY dependencies don't matter and aren't resolved.
nothing ends up building bats and so test fails.

https://stackoverflow.com/questions/13852535/makefile-dependencies-dont-work-for-phony-target